xchange-core/src/main/java/org/knowm/xchange/currency/CurrencyPair.java

```diff
@@ -405,6 +407,7 @@ public CurrencyPair(String baseSymbol, String counterSymbol) {
   /**
    * Parse currency pair from a string in the same format as returned by toString() method - ABC/XYZ
    */
+  @JsonCreator
```

Comment:
I think we should also add `@JsonValue` to the `toString` method in this class

Reply:
Removed the custom converter, since it's no longer necessary